Misc/NEWS.d/next/Library/2021-02-22-22-54-40.bpo-43295.h_ffu7.rst

```diff
@@ -0,0 +1,10 @@
+Previously, datetime.strptime would match 'z' with the format string '%z'
```

Comment:
This is a very good summary of the issue, but it is long for a news stub. They're usually 1 or two lines like this: https://docs.python.org/3/whatsnew/changelog.html

How about:

~~~
Previously, :meth:`datetime.datetime.strptime` would fail with `IndexError` when the string `'z'`
was found for the `%z` format specifier. This has been fixed to properly raise `ValueError` instead.
~~~

(You'll want to check to make sure that the `:meth:` directive works correctly, I may have made a mistake with that.
A BPO link is automatically generated, and users who want more information can click on it for an MWE and such.